stragula.less

```diff
@@ -229,9 +227,16 @@ table a:not(.btn),
 
 @media only screen and (max-width: 480px) {
 	.dropdown-content {
-		left: -100% !important;
-		right: auto !important;
-		column-gap: 0px !important;
+		left: -100%;
+		right: auto;
+		column-gap: 0px;
+	}
+}
+
+@media only screen and (min-width: 480px) {
+	.dropdown-content {
+		left: auto;
+		right: 0;
 	}
 }
 
```

Comment:
@ivanhercaz Here we have conflicting code for the same media definition. I do not think this will work.